### PR TITLE
Allowed inline empty class body

### PR DESCRIPTION
**Pull request by vojtech-dobes**
/cc @vysinsky

### CODE REVIEW

File: DotBlue/tests/ClassDeclarationSniff.phpt [hunk 1]
```diff
@@ -36,4 +36,9 @@ $tester->setFile('ClassDeclaration_5')
 	->onLine(7)
 	->isFixable();
 
+$tester->setFile('ClassDeclaration_6')
+	->setSniff('Classes.ClassDeclaration')
+	->expectMessage('Both opening and closing brace must be on same line as declaration in case of empty body.')
+	->onLine(9);
```
Comment (vysinsky) [inline DotBlue/tests/ClassDeclarationSniff.phpt]: Isn't actual error on line 7?

**Comment by vysinsky** [comment]
Also could you please update README? :)

**Comment by vojtech-dobes** [comment]
> Also could you please update README? :)

Done :).

File: DotBlue/tests/ClassDeclarationSniff.phpt (hunk 1 above)
Comment (vojtech-dobes) [inline DotBlue/tests/ClassDeclarationSniff.phpt]: Thanks, yep. I ran locally only CS and not tests :).

**Comment by vysinsky** [comment]
Thanks!

**Pull request merged**